browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -5657,6 +5667,24 @@ L.CanvasTileLayer = L.Layer.extend({
 		}
 	},
 
+	_onWorkerMessage: function(e) {
+		switch (e.data.message) {
+		case 'decompress':
+			var params = this._workerMap.get(e.data.tile.id);
+			this._workerMap.delete(e.data.tile.id);
+			if (params.isKeyframe)
+				--params.tile.hasPendingKeyframe;
+			else
+				--params.tile.hasPendingDelta;
+			this._applyDelta(params.tile, e.data.tile.rawDelta, e.data.tile.deltas, params.isKeyframe, params.wireMessage, false);
```

Comment:
Whoops, small fixup needed here - `_applyDelta` only has 5 parameters, this 6th `false` is a hangover from earlier where a `needsUnpremultiply` parameter was temporarily added.